server/viewer_embed.go

```diff
@@ -0,0 +1,19 @@
+// +build !noViewer,go1.16
```

Comment:
Is there a reason why this is an OR and not an AND like embed.go.

`// +build !noViewer go1.16`?

Reply:
It should be.

I had to look up the syntax for build constrains/tags. According to the docs and some help from the IDE, the rules are. 

A build constraint is **_evaluated as the OR of space-separated_** options. Each option evaluates as the **AND of its comma-separated** terms.

https://pkg.go.dev/cmd/go#hdr-Build_constraints

Reply:
Okay, yeah I was confused. I had to read it several times.